### ISSUE TITLE
Use git fetch --prune

### ISSUE THREAD

**Issue by andreas-schwab** [openSUSE/obs-service-tar_scm#184]
From git://sourceware.org/git/glibc
 + b570759fc5...d8972c5030 hjl/cet/pr21598            -> hjl/cet/pr21598  (forced update)
 + 57f2b07c55...2a355dbc47 hjl/cet/property           -> hjl/cet/property  (forced update)
error: cannot lock ref 'refs/heads/hjl/pr21913/master': 'refs/heads/hjl/pr21913' exists; cannot create 'refs/heads/hjl/pr21913/master'
 ! [new branch]            hjl/pr21913/master         -> hjl/pr21913/master  (unable to update local ref)
   cd65836b5f..36ada5f681  master                     -> master
 * [new branch]            origin/release/2.25/master -> origin/release/2.25/master
   05dcf1ea88..5b3f978ccf  release/2.25/master        -> release/2.25/master

**Comment by aspiers** [comment]
Yep, using `--prune` sounds like a good idea to work around this corner case.
